app/database/schema.test.js

```diff
@@ -65,6 +67,7 @@ describe('app/database/schema.js', () => {
 				added_time: time,
 				is_active: 1,
 				name: 'xxx',
+				url: 'xxx,
```

Comment:
Travis CI complains to missing `'`

 SyntaxError: /home/travis/build/pipedrive/graphql-schema-registry/app/database/schema.test.js: Unterminated string constant (34:9)
      32 | 				is_active: 1,
      33 | 				name: 'xxx',
    > 34 | 				url: 'xxx,